src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,279 @@
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-1-item-1"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-1-item-2"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-1-item-3"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                </ul>
+              </div>
+            </div>
+            <div class="o-footer__social-media">
+              <slot
+                name="column-2-social-title"
+                class="o-footer__social-media-title"
+              ></slot>
+              <ul class="o-footer__social-media-list">
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-0"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-1"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-2"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-3"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-4"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-5"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </axa-container>
+      </footer>
+    `;
+  }
+
+  updated() {
+    this._removeEmptyListElements();
+  }
+
+  _removeEmptyListElements() {
+    this.shadowRoot
+      .querySelectorAll('.o-footer__main-content-panel-list-item')
+      .forEach(el => {
+        const label = el.querySelector('slot').assignedNodes()[0];
```

Comment:
https://developer.mozilla.org/de/docs/Web/API/HTMLSlotElement/assignedNodes
Assigned nodes seems not to be IE compatible. Does one of our polyfill take care of it?

Reply:
Yes